modules/gis/src/lib/geojson-to-binary.js

```diff
@@ -16,31 +16,28 @@ function firstPass(features) {
   let polygonPositions = 0;
   let polygonObjects = 0;
   let polygonRings = 0;
-  let maxCoordLength = 2;
+  const coordLengths = new Set();
```

Comment:
Interesting approach.

Reply:
I changed it because I thought it might be helpful to know not only that the max coordinate length is 3, but also that there are mixed coordinates. Having the `Set` would tell me both pieces of information. That said, right now I don't think I use that information. 

Do you think there's a performance hit of using a `Set`?

Reply:
> Do you think there's a performance hit of using a Set?

Nah, its fine.